framework/interface/templates/plugin_report.html

```diff
@@ -41,20 +41,21 @@ <h4 id="header_badge_{{ test_code }}"></h4>
-                                <label id="{{ pkey }}_5" class="btn {% if user_rank == 5 %}active btn-critical{% end %}" title="Critical"><input type="radio" id="5" onchange="updatePluginInfoAndPatch('{{ pkey }}', '{{ poutput["plugin_group"] }}','{{ poutput["plugin_type"] }}','{{ test_code }}', '5');"/><i class="fa fa-bomb"></i></label>
+                                <label id="{{ pkey }}_0" class="btn {% if user_rank == 0 %}active{% end %} rank" title="No ranking" data-code="{{ test_code }}"><input type="radio" id="0" onchange="updatePluginInfoAndPatch('{{ pkey }}', '{{ poutput["plugin_group"] }}','{{ poutput["plugin_type"] }}','{{ test_code }}', '0');"/><i class="fa fa-thumbs-up"></i></label>
+                                <label id="{{ pkey }}_1" class="btn rank {% if owtf_rank == 1 %}active btn-success" style="opacity: 0.7{% end %}" title="Informational" data-code="{{ test_code }}"><input type="radio" id="1" onchange="updatePluginInfoAndPatch('{{ pkey }}', '{{ poutput["plugin_group"] }}','{{ poutput["plugin_type"] }}','{{ test_code }}', '1');"/><i class="fa fa-info-circle"></i></label>
+                                <label id="{{ pkey }}_2" class="btn rank{% if owtf_rank == 2 %}active btn-info" style="opacity: 0.7{% end %}" title="Low" data-code="{{ test_code }}"><input type="radio" id="2" onchange="updatePluginInfoAndPatch('{{ pkey }}', '{{ poutput["plugin_group"] }}','{{ poutput["plugin_type"] }}','{{ test_code }}', '2');"/><i class="fa fa-exclamation-circle"></i></label>
+                                <label id="{{ pkey }}_3" class="btn rank{% if owtf_rank == 3 %}active btn-warning" style="opacity: 0.7{% end %}" title="Medium" data-code="{{ test_code }}"><input type="radio" id="3" onchange="updatePluginInfoAndPatch('{{ pkey }}', '{{ poutput["plugin_group"] }}','{{ poutput["plugin_type"] }}','{{ test_code }}', '3');"/><i class="fa fa-warning"></i></label>
+                                <label id="{{ pkey }}_4" class="btn rank{% if owtf_rank == 4 %}active btn-danger" style="opacity: 0.7{% end %}" title="High" data-code="{{ test_code }}"><input type="radio" id="4" onchange="updatePluginInfoAndPatch('{{ pkey }}', '{{ poutput["plugin_group"] }}','{{ poutput["plugin_type"] }}','{{ test_code }}', '4');"/><i class="fa fa-bell"></i></label>
+                                <label id="{{ pkey }}_5" class="btn {% if user_rank == 5 %}active btn-critical{% end %} rank" title="Critical" data-code="{{ test_code }}"><input type="radio" id="5" onchange="updatePluginInfoAndPatch('{{ pkey }}', '{{ poutput["plugin_group"] }}','{{ poutput["plugin_type"] }}','{{ test_code }}', '5');"/><i class="fa fa-bomb"></i></label>
```

Comment:
Keep consistency and move `rank` at the beginning. In fact, I think this one is more logical. Could you move all `rank` attributes at the end of the class items? Like you did in this line